lib: nrf_modem: increase minimum NRF_MODEM_LIB_SHMEM_RX_SIZE (#10038)
nrfconnect/sdk-nrf: doc/nrf/releases/release-notes-changelog.rst

Modem libraries
---------------

* :ref:`nrf_modem_lib_readme` library:
  * Updated:

>     * The minimal value for ``CONFIG_NRF_MODEM_LIB_SHMEM_TX_SIZE`` has been updated to meet the requirements for modem firmware 1.3.4.

Reviewer: You mean `RX` ?

Author: Thanks, updated.

> * :ref:`nrf_modem_lib_readme` library:
>   * Updated:
> 
>     * The minimal value for ``CONFIG_NRF_MODEM_LIB_SHMEM_RX_SIZE`` has been updated to meet the requirements for modem firmware 1.3.4.

Reviewer 2: ```suggestion
* :ref:`nrf_modem_lib_readme` library:

  * Updated:

    * The minimal value of :kconfig:option:`CONFIG_NRF_MODEM_LIB_SHMEM_RX_SIZE` to meet the requirements of modem firmware 1.3.4.
```
